md/moves.py

```diff
@@ -56,3 +56,45 @@ def n_accepted(self):
     @property
     def n_proposed(self):
         return np.sum(self.n_proposed_by_move)
+
+
+from scipy.special import logsumexp
+
+
+class MultipleTryMetropolis:
+    def __init__(self, K):
+        self.K = K
+        self.n_accepted = 0
+        self.n_proposed = 0
+
+    def batch_propose(self, x: CoordsVelBox):
+        raise NotImplementedError()
+
+    def batch_log_prob(self, proposals):
+        raise NotImplementedError()
```

Comment:
Suggestion: Would expose `batch_propose` and `batch_log_prob` as constructor arguments.